setup.py

```diff
@@ -120,7 +120,7 @@ def write_version_py(filename='refnx/version.py'):
         'include_package_data': True,
         'setup_requires': ['numpy'],
         'install_requires': ['numpy', 'scipy', 'emcee', 'six',
-                             'uncertainties', 'pandas']
+                             'uncertainties', 'pandas', 'pyyaml']
```

Comment:
I don't want to make it a required dependency. Makes it a lot harder for people to install from source (esp. Windows) if there are issues installing from conda/pip (or there's a version problem).